src/components/modules/dashboard/marketplace/ui/pages/MarketplaceOffersList.tsx

```
@@ -46,6 +46,8 @@ export default function MarketplaceOffersList() {
   const [sortOption, setSortOption] = useState("Relevance");
   const [viewMode, setViewMode] = useState<"grid" | "list">("grid");
 
+  const [isFilterOpen, setIsFilterOpen] = useState(true);
```

ESLint

[error] 'isFilterOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)